tools/docker/firecracker-runc-config.json

```diff
@@ -0,0 +1,174 @@
+{
+    "ociVersion": "1.0.1-dev",
+    "process": {
+        "terminal": false,
+        "user": {
+            "uid": 0,
+            "gid": 0
+        },
+        "args": [
+            "/usr/local/bin/firecracker",
+            "--api-sock",
+            "api.socket"
+        ],
+        "env": [
+            "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
+            "TERM=xterm"
+        ],
+        "cwd": "/",
+        "capabilities": {
+            "bounding": [
+                "CAP_AUDIT_WRITE",
+                "CAP_KILL",
+		"CAP_NET_ADMIN",
+                "CAP_NET_BIND_SERVICE"
```

Comment:
When is `CAP_NET_BIND_SERVICE` used?

Reply:
Actually, none of those are needed except for `CAP_NET_ADMIN`. Thanks for catching that.